### PR TITLE
Edits

### PR DESCRIPTION
**Pull request by TresRus**
Some ideas about your CV. Will add comments about some parts.

### CODE REVIEW

File: main.tex [hunk 1]
```diff
@@ -135,7 +140,9 @@
 \NewPart{Software Development Skills}{}
 
 \hang{Python}{
-    Four years of experience, well versed in the standard scientific libraries (NumPy, SciPy, Matplotlib) and Pandas to a lesser extent. Personal projects including a \href{https://github.com/fuverdred/Crossword-Filler}{tool for creating themed crosswords}, web-scraping scripts and a \href{https://github.com/fuverdred/pH-Monitor}{device for monitoring and controlling the pH of soil}.
```
Comment (TresRus) [inline main.tex:138]: I'm not quite sure how to make it better but I think it should be two lists - technology/libraries and projects.
Currently it is a long sentances with cool stuff in the middle.

Another thing is version of Python.

File: main.tex [hunk 2]
```diff
@@ -107,8 +103,17 @@
...
+    \begin{itemize}
+        \setlength\itemsep{0em}
+        \item A novel way of automating a standard experimental technique (\href{https://amt.copernicus.org/articles/13/2785/2020/}{published})
+        \item An updated version of an automated lag time apparatus (\href{https://aip.scitation.org/doi/10.1063/1.1145586}{ALTA}) for ice nucleation studies 
+        \item An environmental chamber for freezing acoustically levitated water droplets.
+    \end{itemize}
+    Some relevant highlights of my PhD work include:
     \begin{itemize}
+        \setlength\itemsep{0em}
         \item A program for detecting freezing droplets from a series of images, including tracking the movement of the droplets, written in Python using OpenCV.
```
Comment (TresRus) [inline main.tex:117]: It will be better to keep it shorter. No idea how to do it. Maybe Saskia will have some ideas.

**Comment by TresRus** [comment]
Overall I think it is a good CV and you have a log of great stuff in it - controllers, math libraries, code examples (projects).

File: main.tex (hunk 1 above)
Comment (fuverdred) [reply main.tex:138]: I see what you mean, I will have a think. Something to make it as obvious as possible that I have experience with standard scientific libraries, and then I can point to some of the stuff I have done.

I didn't think of version, always Python3, 3.6+ to be specific, but I think just saying 3 is enough

File: main.tex (hunk 2 above)
Comment (fuverdred) [reply main.tex:117]: Yes I will have a think, she had some comments on the wording of other areas too

**Pull request merged**